tests/unit/test_loader.py

```diff
@@ -976,25 +976,25 @@ def _verify_globals(self, mod_dict):
         Verify that the globals listed in the doc string (from the test) are in these modules
         '''
         # find the globals
-        global_vars = []
+        global_vars = {}
         for val in six.itervalues(mod_dict):
             # only find salty globals
             if val.__module__.startswith('salt.loaded'):
                 if hasattr(val, '__globals__'):
                     if '__wrapped__' in val.__globals__:
-                        global_vars.append(sys.modules[val.__module__].__dict__)
+                        global_vars[val.__module__] = sys.modules[val.__module__].__dict__
                     else:
-                        global_vars.append(val.__globals__)
+                        global_vars[val.__module__] = val.__globals__
 
         # if we couldn't find any, then we have no modules -- so something is broken
-        self.assertNotEqual(global_vars, [], msg='No modules were loaded.')
+        self.assertNotEqual(global_vars, {}, msg='No modules were loaded.')
 
         # get the names of the globals you should have
         func_name = inspect.stack()[1][3]
         names = next(six.itervalues(salt.utils.yaml.safe_load(getattr(self, func_name).__doc__)))
 
         # Now, test each module!
-        for item in global_vars:
+        for key, item in global_vars.iteritems():
```

Comment:
Just one suggestion:
```python
for item in six.itervalues(global_vars.itervalues):
```
Or even better also rename `item` to `value` to conform to the new data pattern.

Reply:
👍